ci-operator/step-registry/agent-qe/baremetal/install/ove/disconnected/collect-validation-results/agent-qe-baremetal-install-ove-disconnected-collect-validation-results-commands.sh

```
@@ -2,6 +2,10 @@
 
 set -uo pipefail
 
+if [ -f "${SHARED_DIR}/proxy-conf.sh" ] ; then
+    source "${SHARED_DIR}/proxy-conf.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./proxy-conf.sh was not specified as input (see shellcheck -x).

(SC1091)